bindings/node/package-lock.json

```diff
@@ -5324,9 +5324,9 @@
       "dev": true
     },
     "tar": {
-      "version": "6.1.0",
```

Comment:
I have no ideas where it from, I don't see this change in my branch https://github.com/DmitryLukyanov/libmongocrypt/commit/07f6c49e566e8a6d19e088d4562f56e913aed286

Reply:
fixed